backend/redis/sender.c

```diff
@@ -315,6 +315,5 @@ void dbBE_Redis_sender_trigger( dbBE_Redis_context_t *backend )
   args->_looping = 1;
   dbBE_Redis_sender( (void*) args );
   dbBE_Redis_receiver( (void*) args );
-  dbBE_Redis_sender( (void*) args );
```

Comment:
I think this answer the previous comment

Reply:
see above. it had a bad effect on overall performance. The impact of coalescing is much more significant than additional and immediate sender processing after receive might have created new requests.